### PR TITLE
added sepcial tag for 16.04 qglviewer qt4

### PR DESCRIPTION
**Pull request by hemker**

### CODE REVIEW

**Comment by doudou** [comment]
The 'default' entry should be the up-to-date one (as 16.10 is most likely to use the same naming than 16.04)

So, add an entry for all ubuntus between 13.10 and 15.10 with the current default and then change the default.

**Comment by hemker** [comment]
done

File: rock.osdeps [hunk 1]
```diff
@@ -367,9 +367,8 @@ unzip:
 
 libqglviewer:
     ubuntu:
-        '12.04,12.10,13.04': libqglviewer-qt4-dev
-        '16.04': libqglviewer-dev-qt4
-        default: libqglviewer-dev
+        '12.04,12.10,13.04,13.10,14.04,14.10,15.04,15.10': libqglviewer-qt4-dev
```
Comment (doudou) [inline rock.osdeps]: You sure about that one ? The previous default was 'libqglviewer-dev' without the qt4

File: rock.osdeps [hunk 2]
```diff
@@ -367,8 +367,8 @@ unzip:
 
 libqglviewer:
     ubuntu:
-        '12.04,12.10,13.04': libqglviewer-qt4-dev
-        default: libqglviewer-dev
+        '12.04,12.10,13.04,13.10,14.04,14.10,15.04,15.10': libqglviewer-dev
```
Comment (doudou) [inline rock.osdeps]: Let's try again ...

Between 12.04 and 13.04 the name was libqglviewer-qt4-dev
*Before* your change, the default was libqglviewer-dev

So unless you have special information (as e.g. that libqglviewer-dev was valid between 12.04 and 13.04), please don't change the existing mappings. Thanks.

**Comment by jmachowinski** [comment]
superseded by https://github.com/rock-core/rock-package_set/pull/147

**Comment by doudou** [comment]
Yeah. And I should definitely not have merged #147 :(

**Comment by doudou** [comment]
Rebased @hemker patch in #148

**Pull request closed without merging**